### PR TITLE
Create custom types for missing OData types.

### PR DESCRIPTION
**Pull request by SilasKenneth**
## Summary

Give a description of what your pull request does and what platform[s] it targets. Use the *breaking changes* tag if applicable. Breaking changes can include changes to generated code files which will impact API signature or behaviors, and changes to directory structures which may impact DevOp pipelines and scripts.

## Generated code differences

Pull requests that result in changes to generated code and/or directory structures should include example diffs of the resulting changes. This makes code review much easier.

## Command line arguments to run these changes

Provide the command line arguments here so that reviewers can quickly repro the results of changes.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/643)

### CODE REVIEW

**Comment by baywet** [comment]
Hey @SilasKenneth your branch contains a lot of head merge commits. I think this is because your local was behind when you created the PR. Would you mind rebasing just to be sure please?

```Shell
git switch dev
git pull -p
git switch silas/fix-missing-types
git rebase dev
git push -f
```

**Comment by SilasKenneth** [comment]
> Hey @SilasKenneth your branch contains a lot of head merge commits. I think this is because your local was behind when you created the PR. Would you mind rebasing just to be sure please?
> 
> ```shell
> git switch dev
> git pull -p
> git switch silas/fix-missing-types
> git rebase dev
> git push -f
> ```

Hi @baywet 
Tried this but it didn't work. Exploring other options.

File: src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs [hunk 1]
```diff
@@ -80,6 +81,8 @@ public static string GetTypeString(this OdcmType @type)
                 case "Binary":
                 case "Stream":
                     return "\\GuzzleHttp\\Psr7\\Stream";
+                case "Date" or "TimeOfDay" or "Byte":
+                    return $"\\Microsoft\\Graph\\Core\\Models\\{@type.Name}";
```
Comment (Ndiritu) [inline src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs:85]: This PR should be merged to `support/php-2.x.x`.

Comment (Ndiritu) [reply src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs:85]: Also basing off the v2 branch would be easier

**Comment by SilasKenneth** [comment]
Closing this  for #644

**Pull request closed without merging**